cluster/gce/gci/configure-helper.sh

```diff
@@ -737,6 +737,10 @@ function start-kube-apiserver {
     params+=" --enable-garbage-collector=${ENABLE_GARBAGE_COLLECTOR}"
   fi
   if [[ -n "${NUM_NODES:-}" ]]; then
+    # If the cluster is large, increase max-requests-inflight limit in apiserver.
+    if [[ "${NUM_NODES}" -ge 1000 ]]; then
+      params+=" --max-requests-inflight=1500"
```

Comment:
Or in other words - currently it is not possible to change the value of this flag (if you are using GCE) other than by changing default of this flag. So the only risk is that the default will be change - we can definitely leave with it.